sqlglot/expressions.py

```diff
@@ -2649,11 +2650,7 @@ class Update(Expression):
 
 
 class Values(UDTF):
-    arg_types = {
-        "expressions": True,
-        "ordinality": False,
```

Comment:
This didn't seem to be used anywhere in the codebase and no tests failed after getting rid of it.